src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/TemplateLocalFileSystem.cs

```diff
@@ -87,7 +87,7 @@ private void LoadMappingTemplate(DataType dataType)
             }
         }
 
-        private string LoadTemplate(string templatePath)
+        private static string LoadTemplate(string templatePath)
         {
             try
```

Comment:
shall we handle the exception in caller function?

Reply:
The template loading logics are refined in the update. `GetTemplate(string templateName)` may still return null when getting templates for code mapping templates, which can be optional.